src/site/pages/guidelines.md

```diff
@@ -64,77 +64,76 @@ PatternFly follows a two-layer theming system where **global variables** always
 The main reason to have global variables is to maintain consistency. They adhere to the following rules:
 
 - They are prefixed with the word `global` and follow the formula `--pf-global--concept--PropertyCamelCase--modifier--state`.
```

Comment:
Above where it says: `The component itself never has backgrounds, widths, floats or margins.` we give components backgrounds so should we remove that